src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -55,10 +56,18 @@ case class WorkflowDescriptor(id: WorkflowId,
   private val props = sys.props
   private val relativeWorkflowRootPath = s"$name/$id"
   private val log = WorkflowLogger("WorkflowDescriptor", this)
+  val workflowLogDir = workflowOptions.get("workflow_log_dir") recover { case e: IllegalArgumentException =>
```

Comment:
It'd be good if these three didn't just throw since all three could independently be broken